mlperf_logging/mllog/constants.py

```diff
@@ -179,6 +179,7 @@
 STATUS = "status"
 STEP_NUM = "step_num"
 SAMPLES_COUNT = "samples_count"
+SAMPLES_NUM = "samples_num"
```

Comment:
how is this different from the "samples_count" above? is it possible to just use samples_count?